MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -882,6 +893,29 @@ private void ProcessIncomingCharacter(SessionBase session)
             }
         }
 
+        private void RunProgram(string modulePath, string cmdline)
+        {
+            var exe = cmdline.Split(' ')[0];
+            var args = cmdline.Split(' ').Skip(1).ToArray();
+            exe = Path.ChangeExtension(exe, ".EXE");
+
+            exe = _fileUtility.FindFile(modulePath, exe);
```

Comment:
This isn't persisting the path so when I run it locally in my IDE, it's looking in my IDE's current application directory, not the directory of the module. We probably need to pass in `modulePath` to the `MZFile` and persist that down the chain on the opens

This is the error I get when it's trying to establish the `MZFile`

![image](https://user-images.githubusercontent.com/1139047/114954018-a8edc500-9e27-11eb-8e0c-f0bcec1e68fc.png)

Reply:
Nice catch - forgot to send the full path to MZFile